build.gradle

```diff
@@ -1,72 +1,33 @@
 buildscript {
     repositories {
         jcenter()
-        maven {
-            url 'https://maven.google.com/'
-            name 'Google'
-        }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.0'
     }
 }
 
 plugins {
-    id 'com.github.dcendents.android-maven' version '1.5'
+    // Base
+    id 'java'
+
+    // Publishing on bintray
+    id 'maven'
+    id 'maven-publish'
     id 'com.jfrog.bintray' version '1.7'
 }
 
+apply plugin: 'maven'
+
 def app_version = '0.16.1'
 
-apply plugin: 'com.android.library'
-apply plugin: 'maven'
-group = 'org.saltyrtc.tasks.webrtc'
-version = app_version
-
-// Android configuration
-android {
-    compileSdkVersion 27
-    buildToolsVersion "27.0.3"
-
-    defaultConfig {
-        minSdkVersion 14
-        targetSdkVersion 28
-        versionCode 1
-        versionName "1.0"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-    }
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
-    }
-    lintOptions {
-        abortOnError false
-    }
-    sourceSets {
-        main {
-            jniLibs.srcDirs = ['libs']
-        }
-    }
-    packagingOptions {
-        exclude 'META-INF/NOTICE'
-        exclude 'META-INF/LICENSE'
-    }
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
-}
+// Java source & target version
+sourceCompatibility = '1.8'
+targetCompatibility = '1.8'
 
 // In this section you declare where to find the dependencies of your project
 repositories {
     jcenter()
-    maven { url "http://dl.bintray.com/saltyrtc/maven" }
-    maven {
-        url 'https://maven.google.com/'
-        name 'Google'
-    }
+    maven { url "https://dl.bintray.com/terl/lazysodium-maven" }
```

Comment:
I guess there's no such thing as a "test repository", right?

Reply:
I haven't looked into it much but since it's a `testImplementation`, I'd assume it doesn't matter and is not being pulled in?

Reply:
The repository would still remain and could potentially override dependencies, I think. But yes, in practice it doesn't really matter.

Reply:
If ordering matters, I can reverse if you want...

Reply:
Should be fine:

> A project can have multiple repositories. Gradle will look for a dependency in each repository in the order they are specified, stopping at the first repository that contains the requested module.

https://docs.gradle.org/current/userguide/dependency_management_for_java_projects.html#N10621